### ISSUE TITLE
Crop Values textholders

### ISSUE THREAD

**Issue by LazyNCoder** [staxrip/staxrip#1275]
Hello,

On the Project Options, there are cropping values. These boxes are just some plain text box I guess. I wanted to ask you, if possible, please kindly change them to the ones which have that up/down arrow. That you can change the value by mouse scroll/keyboard arrows/... etc.
Just like any other value boxes on the encoder options. This would make my life easier. Thank you

**Comment by LazyNCoder** [comment]
Sorry about that. Just updated to the last version, and it's already implanted. Thank you. Closed.

**Comment by Dendraspis** [comment]
Yes, I think they were replaced when I added the *Manual Threshold* edits.  😁
